cstubs_applicative/cstubs_applicative.ml

```diff
@@ -0,0 +1,113 @@
+open Base
```

Comment:
This is a second copy of a library we're not using at all. Please delete it

Reply:
Done